package.json

```diff
@@ -11,6 +11,7 @@
     "browser": "NODE_ENV=production webpack",
     "browser:dev": "webpack",
     "build": "npm run clean && npm run cjs && npm run browser",
+    "lint": "eslint lib test",
```

Comment:
It would be nice to have another npm script `format` so future contributor can use it to reformat their code before submitting a PR

Reply:
in a second thought this might pollute the changes in the PR, adding a ci step to lint would be better I guess

Reply:
Added linting to test script 🙂